### DISCUSSION TITLE
stick fight

### DISCUSSION THREAD

**Discussion by Zhuyuan0907** [diamond-rip/Eden#81] [Kit creations]
stickfight:
    enabled: true
    ranked: true
    display-name: '&aStick Fight'
    priority: 20
    damage-ticks: 19
    display-icon: |
      rO0ABXNyABpvcmcuYnVra2l0LnV0aWwuaW8uV3JhcHBlcvJQR+zxEm8FAgABTAADbWFwdAAPTGph
      dmEvdXRpbC9NYXA7eHBzcgA1Y29tLmdvb2dsZS5jb21tb24uY29sbGVjdC5JbW11dGFibGVNYXAk
      U2VyaWFsaXplZEZvcm0AAAAAAAAAAAIAAlsABGtleXN0ABNbTGphdmEvbGFuZy9PYmplY3Q7WwAG
      dmFsdWVzcQB+AAR4cHVyABNbTGphdmEubGFuZy5PYmplY3Q7kM5YnxBzKWwCAAB4cAAAAAJ0AAI9
      PXQABHR5cGV1cQB+AAYAAAACdAAeb3JnLmJ1a2tpdC5pbnZlbnRvcnkuSXRlbVN0YWNrdAAFU1RJ
      Q0s=
    description: '[]'
    potion-effects: ''
    armor-loadout: |
      rO0ABXcEAAAABHNyABpvcmcuYnVra2l0LnV0aWwuaW8uV3JhcHBlcvJQR+zxEm8FAgABTAADbWFw
      dAAPTGphdmEvdXRpbC9NYXA7eHBzcgA1Y29tLmdvb2dsZS5jb21tb24uY29sbGVjdC5JbW11dGFi
      bGVNYXAkU2VyaWFsaXplZEZvcm0AAAAAAAAAAAIAAlsABGtleXN0ABNbTGphdmEvbGFuZy9PYmpl
      Y3Q7WwAGdmFsdWVzcQB+AAR4cHVyABNbTGphdmEubGFuZy5PYmplY3Q7kM5YnxBzKWwCAAB4cAAA
      AAJ0AAI9PXQABHR5cGV1cQB+AAYAAAACdAAeb3JnLmJ1a2tpdC5pbnZlbnRvcnkuSXRlbVN0YWNr
      dAANTEVBVEhFUl9CT09UU3NxAH4AAHNxAH4AA3VxAH4ABgAAAAJxAH4ACHEAfgAJdXEAfgAGAAAA
      AnEAfgALdAAQTEVBVEhFUl9MRUdHSU5HU3NxAH4AAHNxAH4AA3VxAH4ABgAAAAJxAH4ACHEAfgAJ
      dXEAfgAGAAAAAnEAfgALdAASTEVBVEhFUl9DSEVTVFBMQVRFc3EAfgAAc3EAfgADdXEAfgAGAAAA
      BHEAfgAIcQB+AAl0AAZkYW1hZ2V0AAZhbW91bnR1cQB+AAYAAAAEcQB+AAt0AANBSVJzcgAPamF2
      YS5sYW5nLlNob3J0aE03EzRg2lICAAFTAAV2YWx1ZXhyABBqYXZhLmxhbmcuTnVtYmVyhqyVHQuU
      4IsCAAB4cP//c3IAEWphdmEubGFuZy5JbnRlZ2VyEuKgpPeBhzgCAAFJAAV2YWx1ZXhxAH4AHwAA
      AAA=
    contents-loadout: |
      rO0ABXcEAAAAJHNyABpvcmcuYnVra2l0LnV0aWwuaW8uV3JhcHBlcvJQR+zxEm8FAgABTAADbWFw
      dAAPTGphdmEvdXRpbC9NYXA7eHBzcgA1Y29tLmdvb2dsZS5jb21tb24uY29sbGVjdC5JbW11dGFi
      bGVNYXAkU2VyaWFsaXplZEZvcm0AAAAAAAAAAAIAAlsABGtleXN0ABNbTGphdmEvbGFuZy9PYmpl
      Y3Q7WwAGdmFsdWVzcQB+AAR4cHVyABNbTGphdmEubGFuZy5PYmplY3Q7kM5YnxBzKWwCAAB4cAAA
      AAN0AAI9PXQABHR5cGV0AARtZXRhdXEAfgAGAAAAA3QAHm9yZy5idWtraXQuaW52ZW50b3J5Lkl0
      ZW1TdGFja3QABVNUSUNLc3EAfgAAc3EAfgADdXEAfgAGAAAABHEAfgAIdAAJbWV0YS10eXBldAAI
      ZW5jaGFudHN0AAtyZXBhaXItY29zdHVxAH4ABgAAAAR0AAhJdGVtTWV0YXQAClVOU1BFQ0lGSUNz
      cgA3Y29tLmdvb2dsZS5jb21tb24uY29sbGVjdC5JbW11dGFibGVCaU1hcCRTZXJpYWxpemVkRm9y
      bQAAAAAAAAAAAgAAeHEAfgADdXEAfgAGAAAAAXQACUtOT0NLQkFDS3VxAH4ABgAAAAFzcgARamF2
      YS5sYW5nLkludGVnZXIS4qCk94GHOAIAAUkABXZhbHVleHIAEGphdmEubGFuZy5OdW1iZXKGrJUd
      C5TgiwIAAHhwAAAAAXNxAH4AHAAAAAFzcQB+AABzcQB+AAN1cQB+AAYAAAACcQB+AAhxAH4ACXVx
      AH4ABgAAAAJxAH4ADHQABlNIRUFSU3NxAH4AAHNxAH4AA3VxAH4ABgAAAANxAH4ACHEAfgAJdAAG
      YW1vdW50dXEAfgAGAAAAA3EAfgAMdAAEV09PTHNxAH4AHAAAABBwcHBwcHBwcHBwcHBwcHBwcHBw
      cHBwcHBwcHBwcHBwcHA=
    game-rules: '{"receiveKitLoadoutBook":true,"deathOnWater":false,"boxing":false,"bed":false,"breakGoal":false,"portalGoal":false,"projectileOnly":false,"hypixelUHC":false,"spleef":false,"healthRegeneration":true,"showHealth":true,"foodLevelChange":true,"point":true,"rankedPoint":false,"resetArenaWhenGetPoint":false,"build":true,"startFreeze":true,"noDamage":true,"instantGapple":false,"enderPearlCooldown":false,"clearBlock":true,"dropItemWhenDie":false,"noFallDamage":true,"giveBackArrow":false,"respawnTime":5,"maximumPoints":5,"newRoundTime":5}'
    kit-match-types: '["SOLO"]'
    kit-extra-item: '[]'

**Comment by HierarchyMan** [comment]
Old config version
And food change is enabled which shouldn't be

Just change those fields and it will be perfect 🙂

**Reply by Zhuyuan0907** [reply]
> Old config version And food change is enabled which shouldn't be
> 
> Just change those fields and it will be perfect 🙂

Thanks for reminding 💯
